src/com/android/calendar/DynamicTheme.java

```diff
@@ -93,8 +133,15 @@ public static String getPrimaryColor(Context context) {
         return Utils.getSharedPreference(context, COLOR_PREF, TEAL);
     }
 
-    private static String getSuffix(String theme) {
+    private static String getSuffix(String theme, Context context) {
```

Comment:
API wise this looks a little strange as you get theme twice (once within context). I would propose to drop the theme argument and call getTheme in here.